packages/client/src/checkout/types/postCheckoutSession.types.ts

```diff
@@ -29,9 +29,14 @@ export type PostCheckoutSessionDataWithPaymentLink = PostCheckoutSessionData & {
   removePurchasedItemsFromBag?: boolean;
 };
 
+export type PostCheckoutSessionDataWithBagId = PostCheckoutSessionData & {
+  bagId: string;
```

Comment:
Can we make use of `Bag['id']` type here?

Reply:
Of course, good catch!